ci/si_pipeline.sh

```diff
@@ -76,7 +76,7 @@ case $CLUSTER_LAUNCH_CODE in
   0)
       "$ROOT_PATH/ci/dataDogClient.sc" "marathon.build.si.$VARIANT.cluster_launch.success" 1
       cp -f "$DOT_SHAKEDOWN" "$HOME/.shakedown"
-      make test
+      timeout --preserve-status -s KILL 1s make test
```

Comment:
so we're killing after `1s`? 🤣 I don't think we're that fast yet

Reply:
We are getting there...